core/src/main/java/com/radixdlt/ledger/StateComputerLedger.java

```diff
@@ -326,7 +327,11 @@ private void commit(LedgerExtension ledgerExtension, VertexStoreState vertexStor
     synchronized (this.commitAndAdvanceLedgerLock) {
       final LedgerProof againstLedgerHeader = this.currentLedgerHeader;
 
-      if (this.headerComparator.compare(nextHeader, againstLedgerHeader) <= 0) {
+      if (nextHeader.getStateVersion() <= againstLedgerHeader.getStateVersion()) {
+        log.trace(
+            "ignoring the ledger extension {} which would not progress the current ledger {}",
```

Comment:
Ignoring*

Reply:
ok, let me wait with this push (let's not cancel the CI)